src/txmempool.h

```diff
@@ -105,7 +98,24 @@ class CTxMemPoolEntry
     uint64_t nSizeWithAncestors;
     CAmount nModFeesWithAncestors;
     int64_t nSigOpCostWithAncestors;
+    size_t nTxCost;            //!< ... and avoid recomputing tx cost (also used for GetTxSize())
+    size_t nModSize;           //!< ... and modified size for priority
+    size_t nUsageSize;         //!< ... and total memory usage
+public:
+    mutable size_t vTxHashesIdx; //!< Index in mempool's vTxHashes
+private:
+    unsigned int entryHeight;  //!< Chain height when entering the mempool
 
+    // Space that is typically used as padding. Commented out so that it doesn't cause extra memory use on different architectures.
+    // char PADDING_FREE_SPACE[2];
+
+    // Bool flags
+    struct CTxMemPoolEntryFlags {
+        int UNUSED_BITS : 6;
```

Comment:
@jeremyrubin size_t is the size of a pointer, which is 32 bits on 32 bits architectures and 64 bits on 64 bits architectures. You're right that the overall alignment is 8 bytes (on my architecture at least).

It seems the flags field becomes 4 bytes anyway, so there is no padding afterwards.